### ISSUE TITLE
Generating a typescript interface with a nullable value not working

### ISSUE THREAD

**Issue by goetzst** [asyncapi/modelina#1596]
#### Describe the bug
When trying to generate a typescript interface from a yaml definition with an attribute that is nullable it is generated as a union type with any instead of null.
Also the behaviour of `type` and `oneOf` differ when declaring the properties type.

#### How to Reproduce
Use the following minimal contract.yaml:
```
asyncapi: 2.6.0
id: urn:com:test

info:
  title: Sample
  version: 0.1.0
  description: This is a string null type sample

servers:
  local:
    url: localhost:1234
    protocol: nats
    description: local NATS JetStream Server


channels:
  tests.test:
    description: Tests the string null type message
    subscribe:
      message:
        $ref: "#/components/messages/TestDto"


components:
  messages:
    TestDto:
      payload:
        $ref: "#/components/schemas/testDto"


  schemas:
    testDto:
      type: object
      required: [ someProp]
      additionalProperties: false
      properties:
        someProp:
          oneOf:
            - type: string
            - type: 'null'
          description: A string
          example: "Hello AsyncAPI"
```
Generate interface via: `asyncapi generate models typescript ./asyncTest/contract.yaml --output ./asyncTest/generated --tsModelType interface --tsExportType named`

The resulting interface will look like this:
```
interface TestDto {
  someProp: string | any;
}
```

Using the following contract:
```
asyncapi: 2.6.0
id: urn:com:test

info:
  title: Sample
  version: 0.1.0
  description: This is a string null type sample

servers:
  local:
    url: localhost:1234
    protocol: nats
    description: local NATS JetStream Server


channels:
  tests.test:
    description: Tests the string null type message
    subscribe:
      message:
        $ref: "#/components/messages/TestDto"


components:
  messages:
    TestDto:
      payload:
        $ref: "#/components/schemas/testDto"


  schemas:
    testDto:
      type: object
      required: [ someProp]
      additionalProperties: false
      properties:
        someProp:
          type: [string, 'null]
          description: A string
          example: "Hello AsyncAPI"
```

The resulting interface will look like this:
```
interface TestDto {
  someProp: string;
}
```

Used asyncapi version: 0.59.2

#### Expected behavior
The generated interface should look like this:
```
interface TestDto {
  someProp: string | null;
}
```

**Comment by jonaslagoni** [comment]
@goetzst I have a feeling this is solved in v2 of Modelina and CLI uses v1 at the moment. [Reached out to codeowners on Slack to figure out how we can upgrade it](https://asyncapi.slack.com/archives/CQVJXFNQL/p1699391986134599).

Since playground is for whatever reason broken, we can't even test it quickly with it (https://github.com/asyncapi/modelina/issues/1595) 🙃

**Comment by goetzst** [comment]
Thanks for the feedback. Do you want me to move the issue over to the asyncapicli project in this case?

**Comment by jonaslagoni** [comment]
@goetzst nah your good 👌 created a PR to update the version, so all it needs is a review

**Comment by jonaslagoni** [comment]
Should be solved as of CLI v1

**Comment by goetzst** [comment]
@jonaslagoni just tested it. Both issues have been solved as of v1.0.0. Thanks a lot!
